src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,17 @@
-export const TodoList = () => {};
+import React from 'react';
+import { Todo } from '../../types';
+import { TodoInfo } from '../TodoInfo';
+
+type Props = {
+  todos: Todo[];
+};
+
+export const TodoList: React.FC<Props> = ({ todos }) => {
+  return (
+    <section className="TodoList">
+      {todos.map((todo: Todo) => (
+        <TodoInfo todo={todo} key={todo.id} />
```

Comment:
Ensure that each `todo` has a unique `id`. React relies on the `key` prop to identify elements, and duplicate keys can lead to rendering issues.